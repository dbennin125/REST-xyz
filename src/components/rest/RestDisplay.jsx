import React from 'react';
import PropTypes from 'prop-types';

import ReactJson from 'react-json-view';

const restDisplay = ({ display }) => {
  return (
    <div>
      <ReactJson src={display} />
    </div>
  );
};

restDisplay.propTypes = {
  display: PropTypes.array
};

export default restDisplay;
