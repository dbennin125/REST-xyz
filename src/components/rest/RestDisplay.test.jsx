import React from 'react';
import { shallow } from 'enzyme';
import RestDisplay from './RestDisplay';

describe('RestDisplay component', () => {
  it('renders RestDisplay', () => {
    const wrapper = shallow(<RestDisplay display={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
