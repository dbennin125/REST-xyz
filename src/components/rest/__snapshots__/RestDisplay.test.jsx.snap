// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestDisplay component renders RestDisplay 1`] = `
<div>
  <t
    collapseStringsAfterLength={false}
    collapsed={false}
    defaultValue={null}
    displayDataTypes={true}
    displayObjectSize={true}
    enableClipboard={true}
    groupArraysAfterLength={100}
    iconStyle="triangle"
    indentWidth={4}
    name="root"
    onAdd={false}
    onDelete={false}
    onEdit={false}
    onSelect={false}
    shouldCollapse={false}
    sortKeys={false}
    src={Array []}
    style={Object {}}
    theme="rjv-default"
    validationMessage="Validation Error"
  />
</div>
`;
